sympy/printing/preview.py

```diff
@@ -13,16 +13,17 @@
 from .latex import latex
 
 __doctest_requires__ = {('preview',): ['pyglet']}
+CALLS_TIMEOUT = 10.0  # Time-out system command calls
 
 
 def _check_output_no_window(*args, **kwargs):
     # Avoid showing a cmd.exe window when running this
     # on Windows
     if os.name == 'nt':
-        creation_flag = 0x08000000 # CREATE_NO_WINDOW
+        creation_flag = 0x08000000  # CREATE_NO_WINDOW
```

Comment:
It seems like your editor has made lots of random changes. I suggest disabling whatever setting does that and then only changing the part that needs to be changed.